.github/workflows/fennica.yml

```diff
@@ -34,14 +34,11 @@ jobs:
           extra-packages:
             any::devtools
             local::.
+            quarto
```

Comment:
I think nowadays it's safer to use any::<package> syntax instead of just writing the package name. More info: 

Extra packages and the any:: prefix

In the example above the any:: prefix for ggplot2 and rcmdcheck tells pak to install these packages from CRAN, unless the local package or one of its dependencies request it from somewhere else. E.g. if the checked package required the development version of ggplot2 from https://github.com/tidyverse/ggplot2 then pak will install it from there.